src/components/MultiHopTrade/components/SharedTradeInput/SharedSettingsPopover.tsx

```diff
@@ -123,8 +123,15 @@ export const SharedSettingsPopover: FC<SharedSettingsPopoverProps> = memo(
       [handleSlippageTypeChange],
     )
 
-    const isHighSlippage = useMemo(() => bnOrZero(slippageAmount).gt(1), [slippageAmount])
-    const isLowSlippage = useMemo(() => bnOrZero(slippageAmount).lt(0.05), [slippageAmount])
+    const isHighSlippage = useMemo(
+      () => slippageAmount && bnOrZero(slippageAmount).gt(1),
+      [slippageAmount],
+    )
+    const isLowSlippage = useMemo(
+      () => slippageAmount && bnOrZero(slippageAmount).lt(0.05),
+      [slippageAmount],
+    )
```

Comment:
Also we should revert these for diff purposes

Reply:
Can we keep them for the sake of simplicity? It's fixing a bug that is causing the width to be way too much for nothing on the base state so it's kinda linked to the initial issue 🙏

Reply:
Oh absolutely, didn't see the additional left-side. Let's keep this guy!